tests/integration/test_authd/force_options/test_authd_force_options.py

```diff
@@ -0,0 +1,201 @@
+
+# Variables
+log_monitor_paths = []
+
+receiver_sockets_params = [(("localhost", 1515), 'AF_INET', 'SSL_TLSv1_2')]
+monitored_sockets_params = [('wazuh-authd', None, True), ('wazuh-db', None, True)]
+receiver_sockets, monitored_sockets, log_monitors = None, None, None  # Set in the fixtures
+
+# Functions
+def validate_authd_logs(logs):
+    for log in logs:
+        log_monitor.start(timeout=AUTHD_KEY_REQUEST_TIMEOUT,
+                          callback=make_callback(log, prefix=AUTHD_DETECTOR_PREFIX,
+                                                 escape=True),
+                          error_message='Expected error log does not occured.')
+
+def get_temp_force_config(param):
+    """
+    Creates a temporal config file.
+    """
+    temp = os.path.join(data_path, 'temp.yaml')
+    force_conf = {'force': {'elements': []}}
+    legacy_force_insert_conf = None
+    for elem in param:
+        if elem == 'force_insert':
+            legacy_force_insert_conf = {'force_insert': {'value': param[elem]}}
+        elif elem == 'disconnected_time':
+            disconnected_time_conf = {'disconnected_time':{'value': 0, 'attributes':[{'enabled':'no'}]}}
+            disconnected_time_conf['disconnected_time']['value'] = param[elem]['value']
+            disconnected_time_conf['disconnected_time']['attributes'][0]['enabled'] = param[elem]['attributes'][0]['enabled']
+            force_conf['force']['elements'].append(disconnected_time_conf)
+        else:
+            force_conf['force']['elements'].append({elem: {'value': param[elem]}})
+
+    with open(configurations_path, 'r') as conf_file:
+        temp_conf_file = yaml.safe_load(conf_file)
+        temp_conf_file[0]['sections'][0]['elements'].append(force_conf)
+        if legacy_force_insert_conf:
+            temp_conf_file[0]['sections'][0]['elements'].append(legacy_force_insert_conf)
+    with open(temp, 'w') as temp_file:
+        yaml.safe_dump(temp_conf_file, temp_file)
+    return temp
+
+
+# Fixtures
+
+@pytest.fixture(scope='module', params=configurations, ids=configuration_ids)
+def get_configuration(request):
+    """
+    Get configurations from the module
+    """
+    return request.param
+
+
+@pytest.fixture(scope='function', params=tests, ids=test_case_ids)
+def get_current_test_case(request):
+    """
+    Get current test case from the module
+    """
+    return request.param
+
+
+@pytest.fixture(scope='function')
+def override_authd_force_conf(get_current_test_case, request):
+    """
+    Re-writes Wazuh configuration file with new configurations from the test case.
+    """
+    test_name = request.node.originalname
+    configuration = get_current_test_case.get('configuration', {})
+    # Configuration for testing
+    temp = get_temp_force_config(configuration)
+    conf = load_wazuh_configurations(temp, test_name, )
+    os.remove(temp)
+
+    test_config = set_section_wazuh_conf(conf[0]['sections'])
+    # Set new configuration
+    write_wazuh_conf(test_config)
```

Comment:
Applying the last comment, this fixture can be generalized and moved to the integration tests conftest file.
In order to do that, create a fixture, `format_configuration` as follows:
```
@pytest.fixture(scope='function')
def format_configuration(get_current_test_case)
```
This, should format and return authd configuration according to the current standard.
Using this structure, generic override_authd_force_conf should be something similar to this:
```
 def override_authd_force_conf(format_configuration, request)
```

Another option is to change _`yaml`_ tcase in order to follow the current standard.